db/schema.rb

```diff
@@ -24,7 +24,7 @@
     t.datetime "created_at",     null: false
     t.datetime "updated_at",     null: false
     t.string   "slug"
-    t.datetime "closes_at"
+    t.datetime "closes_at",      null: false
```

Comment:
This should be fine. There are no events with that field blank, and [the migration](https://github.com/artsy/rsvp-rails/blob/9f7b5efabb3d3165db74f068b2c0e1d6ae0f80e0/db/migrate/20161101185508_add_closes_at_to_events.rb#L3) that introduced it specified `null: false`. So I think this is correct.